operator/manager/rpc2/handler.go

```diff
@@ -26,6 +26,8 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/drone/drone/handler/api/render"
```

Comment:
Organize imports. Move this line below, where the other drone/drone imports are.

Reply:
sorted